app/components/collections/terms_of_use_component.rb

```diff
@@ -9,6 +9,18 @@ def initialize(collection:)
 
     attr_reader :collection
 
-    delegate :default_license, :required_license, :user_can_set_license?, to: :collection
+    delegate :allow_custom_rights_statement?, :custom_rights_statement_source_option,
+      :default_license, :effective_custom_rights_instructions, :provided_custom_rights_statement,
+      :required_license, :user_can_set_license?, to: :collection
+
+    def collection_custom_rights_summary
+      return "Additional terms of use are disabled for this collection" unless allow_custom_rights_statement?
+
+      if custom_rights_statement_source_option == "provided_by_collection"
+        "\"#{provided_custom_rights_statement}\""
+      else
+        "The depositor is allowed to enter their own terms. They will be presented with the following instructions: \"#{effective_custom_rights_instructions}\""
+      end
+    end
```

Comment:
this method felt a bit too display-ish for the model, but could see an argument for moving it there.